src/main/java/com/aws/greengrass/logging/impl/GreengrassLogMessage.java

```diff
@@ -37,7 +43,9 @@
     private Throwable cause;
 
     @JsonIgnore
-    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final ObjectMapper OBJECT_MAPPER =
+            new ObjectMapper().disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
```

Comment:
Let's make sure to disable any other reasonable "FAIL_ON" stuff. We really don't want the serialization to fail unless it absolutely needs to.

Reply:
Looks like for `SerializationFeature` we just have these failure defaults: FAIL_ON_EMPTY_BEANS(true), FAIL_ON_SELF_REFERENCES(true), FAIL_ON_UNWRAPPED_TYPE_IDENTIFIERS(true).
The other two defaults should be fine.